h/views/activity.py

```diff
@@ -165,16 +160,9 @@ def leave(self):
         browser to the search page.
 
         """
-        pubid = self.request.POST['group_leave']
-
-        try:
-            group = (self.request.db.query(models.Group)
-                     .filter_by(pubid=pubid).one())
-        except exc.NoResultFound:
-            raise httpexceptions.HTTPNotFound()
```

Comment:
No longer need to fetch the `Group` from the db itself, or worry about when the group doesn't exist